Misc/NEWS.d/next/Core_and_Builtins/2025-10-29-11-31-59.gh-issue-140729.t9JsNt.rst

```diff
@@ -0,0 +1,2 @@
+Fix: Add __mp_main__ as a duplicate for __main__ for pickle to work in
```

Comment:
This needs to refer the profiler otherwise is not possible to undestand what this is fixing. Also this should say WHAT is fixed not HOW